bettermdptools/utils/plots.py

```diff
@@ -55,7 +63,6 @@ def plot_policy(val_max, directions, map_size, title, show=True):
             yticklabels=[],
             annot_kws={"fontsize": "xx-large"},
         ).set(title=title)
-        img_title = f"Policy_{map_size[0]}x{map_size[1]}.png"
```

Comment:
Thanks! In the future we should also add a save fig option but that can be a future PR :)